jdk/test/java/awt/appletviewer/IOExceptionIfEncodedURLTest/IOExceptionIfEncodedURLTest.java

```diff
@@ -21,15 +21,16 @@
  * questions.
  */
 
-/*
-  test
-  @bug 6193279
-  @summary REGRESSION: AppletViewer throws IOException when path is encoded URL
-  @author Dmitry Cherepanov: area=appletviewer
-  @run compile IOExceptionIfEncodedURLTest.java
-  @run main IOExceptionIfEncodedURLTest
-  @run shell IOExceptionIfEncodedURLTest.sh
-*/
+/**
+ * @test
```

Comment:
This will cause the next issue: https://bugs.openjdk.org/browse/JDK-8186259
It will be good to include that fix as well to prevent that regression

Reply:
no you can add a link to that issue to this PR, using "/issue" commands, see:
https://wiki.openjdk.org/pages/viewpage.action?pageId=56526014#PullRequestCommands-/issue